components/cloudtables/cloudtables.app.mjs

```diff
@@ -1,11 +1,92 @@
+import { axios } from "@pipedream/platform";
+import { stringify } from "qs";
+
 export default {
   type: "app",
   app: "cloudtables",
-  propDefinitions: {},
+  propDefinitions: {
+    datasetID: {
+      type: "string",
+      label: "Data set",
+      description: "The Data Set ID",
+      async options() {
+        return this.getDataSets(this);
+      },
+    },
+    rowID: {
+      type: "integer",
+      label: "Row ID",
+      description: "Row ID",
+    },
+  },
   methods: {
+    async _makeRequest($, opts) {
+      if (!opts.headers) opts.headers = {};
+      opts.headers["user-agent"] = "@PipedreamHQ/pipedream v0.1";
+      if (!opts.method) opts.method = "get";
+      const { path } = opts;
+      delete opts.path;
+      opts.url = `https://${this.$auth.subdomain}.cloudtables.io/api/1${path[0] === "/"
```

Comment:
I see you are using query parameters here. Axios has a property called `params`, you can create an object with your query params and pass it to this param.
Take a look on `getBlogPosts` at `blogger.app.mjs` for reference